code/modules/projectiles/projectile/special.dm

```diff
@@ -20,6 +20,23 @@
 	empulse(target, 0, 0)
 	return 1
 
+/obj/item/projectile/ion/strong
+	damage = 35
+	damage_type = TOX
+	nodamage = 0
+	weaken = 2

```

Comment:
I think I was trying to get something going to how an EMP actually works. In hindsight I should probably put it back to brute damage instead.